pkg/storage/fs/owncloud/upload.go

```diff
@@ -423,12 +426,12 @@ func (fs *ocfs) AsTerminatableUpload(upload tusd.Upload) tusd.TerminatableUpload
 
 // Terminate terminates the upload
 func (upload *fileUpload) Terminate(ctx context.Context) error {
-	if err := os.Remove(upload.infoPath); err != nil {
+	if err := os.RemoveAll(upload.infoPath); err != nil {
 		if !os.IsNotExist(err) {
```

Comment:
then you can remove the IsNotExist check as well, because RemoleAll does not return an error in that case

Reply:
I can just revert this. The IsNotExist check is not present in localfs and this reported errors for zero-size files in that case. I'll just add it there instead.